sdk/repo/verify.go

```diff
@@ -232,10 +264,32 @@ func VerifySync(name string) error {
 			}
 
 			if !found {
-				plog.Errorf("Project dir %s at %q, expected %s",
+				plog.Errorf("Project dir %s at branches %q, expected %s (ignore the refs/heads/ prefix)",
 					project.Path, branches, project.Revision)
 				result = VerifyError
 			}
+		} else if strings.HasPrefix(project.Revision, "refs/tags/") {
+			tags, err := manifest.projectTags(project)
+			if err != nil {
+				return err
+			}
+
+			found := false
+
+			for _, tag := range tags {
+				tagRef := "refs/tags/" + tag
+
+				if tagRef == project.Revision {
```

Comment:
I don't understand why you go through the trouble of removing the tags/refs/ part in the other function if you're going to add it here... Do you need the isolated tagname somewhere?

Reply:
In the normal case this is just `"tag: v0.1.0"` and I didn't want to add it there. But maybe remove it here from `project.Revision` once in beginning.